### PR TITLE
Error tests

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- Test system modified

### CODE REVIEW

File: Test/plot_error_test.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 >>> ax = cm.plot(plot_lib='seaborn')
 Traceback (most recent call last):
     ...
-pycm.pycm_error.pycmPlotError: Failed to import seaborn module. Please install it using: `pip install seaborn`.
+pycm.pycm_error.pycmPlotError: Failed to import matplotlib module. Please install it using: `pip install matplotlib`.
```
Comment (sadrasabouri) [inline Test/plot_error_test.py:25]: This shouldn't be changed, right?
it requested the `seaborn` in line `22`)

**Review by sadrasabouri — COMMENTED** [review]
Just a minor comment.
I guess since it's the continue on Arash's PR we also don't have `Changelog.md` changes, right?

**Comment by sepandhaghighi** [comment]
> * Test system modified

Yes, as you can see, `Test system modified` has already been added to `CHANGELOG.md`.

**Comment by sepandhaghighi** [comment]
> Just a minor comment. I guess since it's the continue on Arash's PR we also don't have `Changelog.md` changes, right?


Yes, as you can see, `Test system modified` has already been added to `CHANGELOG.md`.

File: Test/plot_error_test.py (hunk 1 above)
Comment (sepandhaghighi) [reply Test/plot_error_test.py:25]: You are correct, but this issue is related to our code, not the tests.
As both engines use `cm` and `seaborn` is based on `matplotlib`, we check the `matplotlib` installation before checking the `plot_lib`.

https://github.com/sepandhaghighi/pycm/blob/f320c42733b23816947a8639c1998fb2739cd960/pycm/pycm_obj.py#L1064-L1071

Comment (sadrasabouri) [reply Test/plot_error_test.py:25]: Oh got it.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**